### PR TITLE
Move construction methods from VaRestJsonValue to VaRestSubsystem

### LINKED ISSUE #226: Move constuction functions to the library

### LINKED ISSUE #266: Remove redirectors for future releases

### PR DESCRIPTION
**Pull request by trdwll**
Fixes #226 

Update: Referenced wrong issue.

### CODE REVIEW

File: Source/VaRest/Private/VaRestSubsystem.cpp [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #include "VaRestDefines.h"
 #include "VaRestJsonObject.h"
 #include "VaRestSettings.h"
+#include "VaRestJsonValue.h"
```
Comment (ufna) [inline Source/VaRest/Private/VaRestSubsystem.cpp]: Please check clang-format error: https://teamcity.ufna.dev/buildConfiguration/UfnaDev_VaRest_ClangFormatCheck/496?mode=branches . All plugin files are processed and checked with clang-format-9 (built-in in MSVS 2019).

File: Source/VaRest/Private/VaRestSubsystem.cpp [hunk 2]
```diff
@@ -107,6 +108,73 @@ UVaRestJsonObject* UVaRestSubsystem::ConstructVaRestJsonObject()
 	return NewObject<UVaRestJsonObject>(this);
 }
 
+UVaRestJsonValue* UVaRestSubsystem::ConstructJsonValueNumber(float Number)
+{
+	TSharedPtr<FJsonValue> NewVal = MakeShareable(new FJsonValueNumber(Number));
+
+	UVaRestJsonValue* NewValue = NewObject<UVaRestJsonValue>();
```
Comment (ufna) [inline Source/VaRest/Private/VaRestSubsystem.cpp]: For my mind it's a good idea to set Outer here (and below) to `this`. What do you think about it?

File: Source/VaRest/Private/VaRestSubsystem.cpp (hunk 1 above)
Comment (trdwll) [reply Source/VaRest/Private/VaRestSubsystem.cpp]: Done

File: Source/VaRest/Private/VaRestSubsystem.cpp (hunk 2 above)
Comment (trdwll) [reply Source/VaRest/Private/VaRestSubsystem.cpp]: Done

**Comment by ufna** [comment]
@trdwll , thank again! :)

**Pull request merged**